ansible_collections/arista/cvp/molecule/cv_facts_v3/test_cv_facts_v3.yml

```diff
@@ -0,0 +1,257 @@
+      arista.cvp.cv_facts_v3:
+        facts:
+          - images
+        regexp_filter: "{{IMAGE_NAME}}" # collects specific image facts
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_images['EOS-4.25.4M.swi'] is defined
+
+    - name: Collect task facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - tasks
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks != []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks.total_tasks is defined
+
+    - name: Collect task facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - tasks
+        regexp_filter: "{{TASK_STATUS}}" # filtered by status
+        verbose: long
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks != []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks.total_tasks is defined
+
+    - name: Collect task facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - tasks
+        regexp_filter: "{{MOVE_DEVICE_DATA_EXECUTE.devices_moved.taskIds[0]}}" # filtered by task id
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks != []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks.total_tasks is defined
+
+    - name: Delete config
+      arista.cvp.cv_configlet_v3:
+        configlets: "{{CVP_CONFIGLETS}}"
+        state: absent
+
+    - name: Remove Container
+      arista.cvp.cv_container_v3:
+        topology: '{{CONTAINER}}'
+        state: absent
```

Comment:
we can't do this because the device is under that container `Leafs` so CVP will error out

```
TASK [Remove Container] ********************************************************
fatal: [CloudVision]: FAILED! => changed=false
  msg: 'Unable to delete container Leafs: container not empty - either it has child container(s) or                 some device(s) are attached to it on CVP'
```

but since this is a facts test, I don't think we need to remove the container, otherwise we will need to remove the device first, what do you think @sugetha24 ?

Reply:
fixed